src/tests/timer-test/timer-test.c

```diff
@@ -20,24 +20,29 @@
 #define TASK_1_STACK_SIZE 4096
 #define TASK_1_PRIORITY   101
 
+
 void TimerTestSetup(void);
 void TimerTestTask(void);
 void TimerTestCheck(void);
 
 OS_time_t        StartTime;
 OS_time_t        EndTime;
+uint32           TimerStart[NUMBER_OF_TIMERS];
+uint32           TimerInterval[NUMBER_OF_TIMERS]; 
+
```

Comment:
This looks odd - variable declared without initializer and then defined with initializer.  Possibly a remnant change?  Should remove this addition.

Reply:
This was a typo.  Fixed.